.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -36,6 +36,11 @@ For example:
 
 - [ ] Verify that no errors appear in the JS console
 
+### Offline tests
+<!---
+Add any relevant steps that validate your changes work as expected in a variety of network states e.g. "offline", "spotty connection", "slow internet", etc. Manual test steps should be written so that your reviewer and QA testers can repeat and verify one or more expected outcomes. If you are unsure how the behavior should work ask for advice in the `#expensify-open-source` Slack channel.
```

Comment:
Two quick questions:

1. Are we wrapping slack channels in backticks usually? I think the other places we cite #expensify-open-source is just written plainly. IMO we can link the URL directly to get a similar effect of having it stand out.
>https://expensify.slack.com/archives/C01GTK53T8Q

2. Do we want them to post in #expensify-open-source or to thread up on the bug report in #expensify-bugs? What's more organized 🤔 . I feel like questions will be specific to the bug itself, so I sorta think the latter could be best?

Reply:
Sorry, it's a markdown comment in the PR template you can't make it link anywhere.

>Do we want them to post in #expensify-open-source or to thread up on the bug report in #expensify-bugs? What's more organized 🤔 . I feel like questions will be specific to the bug itself, so I sorta think the latter could be best?

I think what I wrote is what they should do. Isolating it to the bug thread means that only the thread participants will be notified. Anyone should be able to help them resolve their question.

Reply:
Cool, that reasoning makes sense to me. I updated my comment to add:
>so I sorta think the latter could be best if they "share to channel" as well?

But I still think your idea will work well for the utmost visibility. Don't want them to die on the final hill when it comes to testing, so more visibility wins in my/your book. 👍